internal/pkg/manifest/lb_web_svc_test.go

```diff
@@ -236,7 +244,9 @@ func TestLoadBalancedWebSvc_ApplyEnv(t *testing.T) {
 					TaskConfig: TaskConfig{
 						CPU:    aws.Int(2046),
 						Memory: aws.Int(1024),
-						Count:  aws.Int(0),
+						Count: AutoscalingOrTaskCount{
+							TaskCount: aws.Int(0),
```

Comment:
Does mean we can scale to zero?? Or could we always do that?

Reply:
Yeah that's right. We allow users to scale down to 0 and that's basically the reason why we want to use `*int`